.circleci/config.yml

```diff
@@ -180,6 +191,39 @@ commands:
           path: test-results
       - store_artifacts:
           path: test-results
+      - run:
+          name: Get APM Test Agent Trace Check Results
+          when: always
+          command: |
+            set +e  # Disable exiting from testagent response failure
```

Comment:
Putting this code into a file in `scripts/` would reduce the cognitive load incurred by this file.

Reply:
done